packages/wrangler/src/wrangler-config-path.ts

```diff
@@ -0,0 +1,26 @@
+import fs from "node:fs";
+import os from "node:os";
+import path from "node:path";
+import xdgAppPaths from "xdg-app-paths";
+
+function isDirectory(configPath: string) {
+	try {
+		return fs.statSync(configPath).isDirectory();
+	} catch (error) {
+		// ignore error
+		return false;
+	}
+}
+
+export function getWranglerConfigPath() {
```

Comment:
sorry to be annoying about this, thinking we should be even more explicit here, just don't want it confused with wrangler.toml. How do you feel about `getGlobalWranglerConfigPath()`? thanks for considering!

Reply:
I am fine with either, anyone feel particularly strong either way?

Reply:
`getGlobalWranglerConfigPath` isn't much better, but I could probably guess what we're keeping in there, whereas `getXDGConfigPath` would require googling wtf XDG is...

Reply:
I changed my mind, going with Sunil's suggestion since the function is backwards compatible it isn't just able to "get" XDG paths.